### ISSUE TITLE
Testing the model on live videos

### ISSUE THREAD

**Issue by reetsharma** [AdrianNunez/Fall-Detection-with-CNNs-and-Optical-Flow#9]
I have gone through your paper and "temporalneturfd.py" code which was really helpful to build the concept. I am able to build and test the model on URFD dataset. Since I have a requirement to classify the fall in the live videos, would it be possible for you to share the code for streaming in live videos using laptop webcam and then converting it into RGBimages? Once test data get converted into RGB images . Can we use existing model to classify the RGB images rather than Optical flow.
I want to avoid optical flow conversion , since it takes lot of time to get converted. For testing can i used directly RGB images

**Comment by AdrianNunez** [comment]
Hi,

I do not have any code for online learning. In fact, this fall detection method cannot be used for real-time inference, as it requires to compute the optical flow images from the RGB data (which is of course a heavy computational burden). You HAVE TO use optical flow data as input to the network, RGB data cannot be used as input as (1) the network has not been trained on RGB information and (2) because you will get an error due to the mismatch in the channel number (20 vs 3).

If you want to use only RGB data you need to take other approaches like 3D Convolutions or Recurrent Neural Networks (GRU, LSTM).

**Comment by AdrianNunez** [comment]
Hi,

'test_video' is a function I haven't tested, so I cannot promise that works. It should extract a set of features (of an optical flow stack) representing a video (a folder with all its frames). To evaluate on another dataset you will need to modify the code a bit, load your previous model, skip the training but load the test set you want to use so that the evaluation is done using it.
